.drone.yml

```diff
@@ -0,0 +1,63 @@
+---
+kind: pipeline
+type: docker
+name: aarch64 build GCC (native)
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+- name: build
+  image: ubuntu:focal
+  commands:
+  - scripts/ci/apt-install make gcc pkg-config git perl-modules
+  - make -C scripts/ci local
+
+---
+kind: pipeline
+type: docker
+name: aarch64 build CLANG (native)
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+- name: build
+  image: ubuntu:focal
+  commands:
+  - scripts/ci/apt-install make gcc pkg-config git perl-modules
+  - make -C scripts/ci local CLANG=1
+
+---
+kind: pipeline
+type: docker
+name: armhf build GCC (native)
+
+platform:
+  os: linux
+  arch: arm
+
+steps:
+- name: build
+  image: ubuntu:bionic
+  commands:
+  - scripts/ci/apt-install make gcc pkg-config git perl-modules
+  - make -C scripts/ci local
+
+---
+kind: pipeline
+type: docker
+name: armhf build CLANG (native)
+
+platform:
+  os: linux
+  arch: arm
+
+steps:
+- name: build
+  image: ubuntu:bionic
+  commands:
+  - scripts/ci/apt-install make gcc pkg-config git perl-modules
+  - make -C scripts/ci local CLANG=1
```

Comment:
hmm, why is called `CLANG (native)` if it installs `gcc` as a dependency?

Reply:
Ah, that is just copy paste error. The actual packages are installed using our main CI script. I will clean this up.